utm.yaml

```diff
@@ -737,7 +748,10 @@ components:
           - $ref: '#/components/schemas/SubscriptionUssBaseURL'
         notify_for_constraints:
           description: |-
-            True if this operational intent's subscription should trigger notifications when constraints change.  Otherwise, changes in constraints should not trigger notifications.
+            True if this operational intent's subscription should trigger notifications when constraints change.
+            Otherwise, changes in constraints should not trigger notifications.  The scope
+            utm.constraint_processing is required to set this flag true, and a USS performing the constraint
```

Comment:
PUT /dss/v1/operational_intent_references/{entityid} uses this ImplicitSubsrciptionParameters object. However, it does not have utm.constraint_processing in its authority listing (currently it's utm.strategic_coordination and utm.conformance_monitoring_sa). 

Does constraint_processing need to be added to the list to support this requirement?

Reply:
That's a little tricky -- from a minimum authorization standpoint, constraint_processing isn't an alternative to strategic_coordination; a USS must provide only strategic_coordination or strategic_coordination+constraint_processing and only constraint_processing is insufficient.  So, the actual definition would read like:

```yaml
      - Authority:
        - utm.strategic_coordination
      - Authority:
        - utm.strategic_coordination
        - utm.constraint_processing
      - Authority:
        - utm.conformance_monitoring_sa
```

This is a little weird since the second Authority option is a strict subset of the first option, so the first option should cover it -- it's always ok to add extra scopes if the USS wants.  But, it could be useful to convey that this is a recognized combination, so I've added the option as written above.